### PR TITLE
Add Partisia Blockchain namespace

### PR DESCRIPTION
**Pull request by eorloff**

### CODE REVIEW

File: partisia/caip2.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Each network has a unique chain id that can be used to identify the network.
+
+The Partisia Blockchain networks are identified by the `partisia` namespace prefix, followed by the
+chain id of the particular network.
+
+## Syntax
+
+The chain id of Partisia Blockchain is a human-readable identifier of the particular chain. Due to
+the character limitations of CAIP-2 all characters that are not allowed in the specification (
+`[^a-zA-Z0-9]`) are replaced with `_`.
```
Comment (bumblefudge) [inline partisia/caip2.md:42]:
```suggestion
`[^-_a-zA-Z0-9]`) are replaced with `_`.
```

File: partisia/caip2.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+networks.
+Each network has a unique chain id that can be used to identify the network.
+
+The Partisia Blockchain networks are identified by the `partisia` namespace prefix, followed by the
+chain id of the particular network.
+
+## Syntax
+
+The chain id of Partisia Blockchain is a human-readable identifier of the particular chain. Due to
+the character limitations of CAIP-2 all characters that are not allowed in the specification (
```
Comment (bumblefudge) [inline partisia/caip2.md:41]:
```suggestion
the character limitations of CAIP-2 all characters that are not allowed in the CAIP-2 characterset (i.e.,
```

File: partisia/caip2.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+# Request
+curl -s https://reader.partisiablockchain.com/blockchain/chainId
+
+#Response
+{
+  "chainId": "Partisia Blockchain"
+}
+```
+
+To convert this to a valid CAIP-2 identifier one need to replace any characters not allowed by the
```
Comment (bumblefudge) [inline partisia/caip2.md:62]:
```suggestion
To convert this to a valid CAIP-2 identifier one need only replace any characters not allowed by the
```

**Comment by eorloff** [comment]
@bumblefudge Thanks for the great feedback. I have applied your suggestions.

File: partisia/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+namespace-identifier: partisia
+title: Partisia Blockchain
+author: Partisia Blockchain Foundation
+status: Draft
+type: Informational
+created: 2024-10-22
+requires: [ "CAIP-2" ]
```
Comment (bumblefudge) [inline partisia/README.md:8]:
```suggestion
requires: CAIP-2
```

**Review by obstropolos — APPROVED** [review]

**Pull request merged**